### DISCUSSION TITLE
New fieldtype for selecting an emoji

### DISCUSSION THREAD

**Discussion by blue-dreamer** [ExpressionEngine/ExpressionEngine#3103] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
The emoji tags is useful but it isn't client friendly when you want to use them for selecting one to use in an entry.

**Describe the solution you'd like**
A new fieldtype with emoji picker. The selected/current emoji appears as an icon in the publish field. This gives authors a really easy way to select one.

Possible enhancements:
- ability to restrict what emojis are available per field instance - not essential but could have uses.
- ability to add all HTML entities and symbols

**Comment by intoeetive** [comment]
Not sure if that eliminates this FR, but are you aware that on PC you can press Windows + period (.) key combination and that will give you emoji popup, right? Not sure what's the combination on Mac 

![image](https://user-images.githubusercontent.com/752126/154419247-fdf1a88d-ddb4-4a39-ae1c-51693b32e8a4.png)

**Comment by blue-dreamer** [comment]
Now that's new to me, Firefox and Safari, at least, have a menu option Edit > Emojis

I can see that's an relatively easy way to get an emoji which is great. I suppose you could use a text field limited to 1 character to insert the emoji, even though the character is quiet small so it's hard to see what it is. I can see possible issues with less savvy clients because it requires using the browser menu or system shortcut (few clients I know will remember those).

So your suggestion gives an option but I still think a dedicated field would be a lot easier for clients.
